bootstrap

```diff
@@ -1,76 +0,0 @@
-#!/bin/sh
-
-# Script to execute to initialize xmlwrapp build system after checking out
-# pristine sources from a version control system: this script creates all
-# generated files which are needed for the build but not stored under version
-# control.
-
-# Copyright (C) 2005, 2006, 2007, 2008 Vadim Zeitlin
-# All Rights Reserved
-#
-# Redistribution and use in source and binary forms, with or without
-# modification, are permitted provided that the following conditions
-# are met:
-#
-# 1. Redistributions of source code must retain the above copyright
-#    notice, this list of conditions and the following disclaimer.
-# 2. Redistributions in binary form must reproduce the above copyright
-#    notice, this list of conditions and the following disclaimer in
-#    the documentation and/or other materials provided with the
-#    distribution.
-# 3. Neither the name of the Author nor the names of its contributors
-#    may be used to endorse or promote products derived from this software
-#    without specific prior written permission.
-#
-# THIS SOFTWARE IS PROVIDED BY THE AUTHOR AND CONTRIBUTORS ``AS IS''
-# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED
-# TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A
-# PARTICULAR PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE AUTHOR
-# OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
-# SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
-# LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF
-# USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
-# AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
-# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT
-# OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
-# SUCH DAMAGE.
-
-# $Id$
-
-if [ ! -f configure.ac -o ! -f Makefile.am -o ! -f xmlwrapp.pc.in ]; then
-    echo "Please run this script from the xmlwrapp source directory."
-    exit 2
-fi
-
-case "$(uname -s)" in
-    Darwin*)  LIBTOOLIZE=glibtoolize ;;
-    *)        LIBTOOLIZE=libtoolize ;;
-esac
-
-run_optional_tool() {
-    dir=$1
-    tool=$2
-    shift 2
-    if command -v $tool >/dev/null 2>&1 ; then
-        (cd $dir && $tool $*)
-    else
-        echo "   (skipped, $tool not available)"
-    fi
-}
-
-# Use --copy with automake to allow simultaneous use on windows under mingw and
-# cygwin platforms. Symlinking of files under mingw does not work out for
-# cygwin and vice-versa.
-echo "Setting up build system for xmlwrapp:"
-echo " - aclocal " && aclocal && \
-echo " - libtoolize " && $LIBTOOLIZE --copy --automake && \
-echo " - autoconf " && autoconf && \
-echo " - automake " && automake --add-missing --copy && \
-echo " - doxygen "  && run_optional_tool docs doxygen && \
-echo " - bakefile " && run_optional_tool platform/Win32 bkl xmlwrapp.bkl && \
```

Comment:
What about this bakefile bit? MSVS projects should be in the tarball, I think, so it's either running it as part of `make dist` or committing projects into the repository (after switching to `msvs`).

Reply:
Oops, thanks for noticing this. I'll add a `dist-hook` to run bkl for now and will also add `make dist` to the CI builds to ensure that it doesn't get broken again.